### PR TITLE
Release 2.43.0-rc.0

### PR DESCRIPTION
**Pull request by roidelapluie**

### CODE REVIEW

**Review by beorn7 — APPROVED** [review]
Maybe mention https://github.com/prometheus/prometheus/pull/10991 and how to use it somewhere? (E.g. in free-form text before the formal changelog lines?) In that way, we might get more traction of people trying it out.)

**Comment by roidelapluie** [comment]
Can you review the text?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,32 @@
 # Changelog
 
+## 2.43.0-rc.0 / 2023-03-09
+
+We are working on some performance improvements in Prometheus, which are
+available behind the go tag `stringlabels` (therefore are not shipped in the
```
Comment (juliusv) [inline CHANGELOG.md:6]:
```suggestion
only built into Prometheus when compiling it using the Go tag `stringlabels` (therefore they are not shipped in the
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,32 @@
...
+* [FEATURE] Promtool: Add HTTP client configuration to query commands. #11487
+* [FEATURE] Scrape: Add `include_scrape_configs` to include scrape configs from different files. #12019
+* [FEATURE] HTTP client: Add `no_proxy` to exclude URLs from proxied requests. #12098
+* [FEATURE] HTTP client: Add `proxy_from_enviroment` to read proxies from env variables. #12098
+* [ENHANCEMENT] API: Add support for setting lookback delta per query via the API. #12088
+* [ENHANCEMENT] API: Change HTTP status code from 503/422 to 499 if a request is canceled. #11897
+* [ENHANCEMENT] Scrape: Allow exemplars for all metric types. #11984
+* [ENHANCEMENT] TSDB: Add metrics for head chunks and WAL folders size. #12013
+* [ENHANCEMENT] TSDB: Automatically remove incorrect snapshot with index that is ahead of WAL. #11859
+* [ENHANCEMENT] TSDB: Improve Prometheus parser error outputs to be more comprehensive. #11682
```
Comment (juliusv) [inline CHANGELOG.md:24]: comprehensive or comprehensible? I assume the latter, right?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,32 @@
...
+* [ENHANCEMENT] UI: Scope `group by` labels to metric in autocompletion. #11914
+* [BUGFIX] Scrape: Fix `prometheus_target_scrape_pool_target_limit` metric not set before reloading. #12002
+* [BUGFIX] TSDB: Correctly update chunks removed and chunks metrics when reading WAL. #11858
```
Comment (juliusv) [inline CHANGELOG.md:27]: I would reference the actual metric names here, not what they are called in the Go code.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,32 @@
...
+* [BUGFIX] TSDB: Observe delta in seconds `prometheus_tsdb_sample_ooo_delta`. #12004
```
Comment (juliusv) [inline CHANGELOG.md:28]: The PR's description here was a bit lacking, how about:

```suggestion
* [BUGFIX] TSDB: Use the correct unit (seconds) when recording out-of-order append deltas in the `prometheus_tsdb_sample_ooo_delta` metric. #12004
```

**Comment by roidelapluie** [comment]
Thanks @juliusv for the review, I have addressed your feedback

**Review by beorn7 — APPROVED** [review]

**Review by kakkoyun — COMMENTED** [review]
LGTM

**Pull request merged**